### PR TITLE
M0078DDG-441 Remove Studio as a light version of the SDK

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**
Remove Studio as a light version of the SDK.

### CODE REVIEW

**Comment by frivieremicroej** [comment]
@tdelhomenie-microej PR updated with only relevant changed of your commits for immadiate merge

Your original development branch backuped here: https://github.com/MicroEJ/docs/tree/feature/M0078DDG-441_sdk-studio_backup_tde

File: overview/index.rst [hunk 1]
```diff
@@ -3,12 +3,12 @@
 Overview
 ========
 
-MicroEJ offers a comprehensive toolset to build the embedded software of
-a device. The toolset covers two levels in device software development:
+MicroEJ SDK offers a comprehensive toolset to build the embedded software of
```
Comment (tdelhomenie-microej) [inline overview/index.rst:6]: Shouldn't we remove `MicroEJ` here ?

Comment (frivieremicroej) [reply overview/index.rst:6]: I preferred to introduce with "MicroEJ SDK", then use "the SDK" in the rest of the documentation

Comment (tdelhomenie-microej) [reply overview/index.rst:6]: ok

**Review by frivieremicroej — APPROVED** [review]

**Pull request merged**